src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
```

Comment:
On my opinion, its better set for card align-items: center, so that u dont need to write for all elements you want to center margin: 0 auto.
For example if you will add new elements in card, you will also need to write for them margin: 0 auto, but if you have align-items: center for card, they will be centered by default